.github/workflows/node.js.yml

```diff
@@ -5,9 +5,9 @@ name: Node.js CI
 
 on:
     push:
-        branches: [master]
+        branches: [master, rebranding]
     pull_request:
-        branches: [master]
+        branches: [master, rebranding]
```

Comment:
Your branch is called `gemini-testing:rebranding`, so the auto-checks are still not launching